package.json

```diff
@@ -15,11 +15,11 @@
   "homepage": "https://github.com/factiverse/storybook-design-system#readme",
   "scripts": {
     "generate": "node ./util/create-component",
-    "storybook": "start-storybook -p 6006 -s public",
+    "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook",
-    "build-storybook-docs": "build-storybook  --docs",
+    "build-storybook-docs": "build-storybook --docs",
     "chromatic": "npx chromatic --project-token=e8130cbb59b8 --exit-once-uploaded",
-    "deploy-storybook": "storybook-to-ghpages",
+    "deploy": "storybook-to-ghpages",
```

Comment:
deploy command deploys to GitHub pages in other repos, should be called the same here.